src/pages/dho/Configuration.vue

```diff
@@ -245,209 +373,101 @@ export default {
-
-      //       exploreBackgroundImage: this.form.exploreBackgroundImage ? this.form.exploreBackgroundImage : defaultSettings.exploreBackgroundImage,
-      //       exploreTitle: this.form.exploreTitle ? this.form.exploreTitle : defaultSettings.exploreTitle,
-      //       exploreParagraph: this.form.exploreParagraph ? this.form.exploreParagraph : defaultSettings.exploreParagraph,
-
-      //       daoTitle: title,
-      //       ...(hasURLChanged ? { daoUrl: url } : {}),
-      //       proposalsCreationEnabled: form.proposalsCreationEnabled ? 1 : 0,
-      //       membersApplicationEnabled: form.membersApplicationEnabled ? 1 : 0,
-      //       removableBannersEnabled: form.removableBannersEnabled ? 1 : 0,
-
-      //       votingAlignmentX100: form.votingAlignmentPercent,
-      //       votingQuorumX100: form.votingQuorumPercent
-
-      //       // msig_approval_amount: 1
-      //     }
-      //   }
-
-      //   // this.state = CONFIGURATION_STATE.CREATE_MULTI_SIG
-
-      //   await this.createSettingsMultisig(data)
-
-      //   this.state = CONFIGURATION_STATE.WAITING
-
-      //   this.initialForm = {
-      //     ...this.form
-      //   }
-      // } catch (e) {
-      //   const message = e.message || e.cause.message
-      //   this.showNotification({ message, color: 'red' })
-      // }
+      try {
+        this.state = CONFIGURATION_STATE.CREATE_MULTI_SIG
+
+        await this.createSettingsMultisig({
+          docId: this.selectedDao.docId,
+          data: settingsMapper(this.dataForSave)
+        })
+
+        this.initialForm = {
+          ...this.form
+        }
+
+        this.state = CONFIGURATION_STATE.WAITING
+      } catch (e) {
+        const message = e.message || e.cause.message
+        this.showNotification({ message, color: 'red' })
+      }
     },
 
     async cancelMultisig () {
-      // this.state = CONFIGURATION_STATE.SAVING
-      await this.cancelSettingsMultisig({ id: this.activeMultisig.id })
-      // this.state = CONFIGURATION_STATE.WAITING
+      try {
+        this.state = CONFIGURATION_STATE.SAVING
+        await this.cancelSettingsMultisig({ id: this.activeMultisig.id })
+        this.state = CONFIGURATION_STATE.WAITING
+      } catch (e) {
+        const message = e.message || e.cause.message
+        this.showNotification({ message, color: 'red' })
+      }
     },
 
     async voteMultisig (approve) {
-      this.state = CONFIGURATION_STATE.SAVING
-      await this.voteSettingsMultisig({ id: this.activeMultisig.id, approve })
-      this.state = CONFIGURATION_STATE.WAITING
+      try {
+        this.state = CONFIGURATION_STATE.SAVING
+        await this.voteSettingsMultisig({ id: this.activeMultisig.id, approve })
+        this.state = CONFIGURATION_STATE.WAITING
+      } catch (e) {
+        const message = e.message || e.cause.message
+        this.showNotification({ message, color: 'red' })
```

Comment:
@arsenijesavic we should pull the colour from some colour pallet, instead of writing strings directly